pkg/bootstrap/config.go

```diff
@@ -325,6 +325,7 @@ func extractRuntimeFlags(cfg *model.NodeMetaProxyConfig) map[string]any {
 		"re2.max_program_size.error_level":           "32768",
 		"envoy.deprecated_features:envoy.config.listener.v3.Listener.hidden_envoy_deprecated_use_original_dst": true,
 		"envoy.reloadable_features.http_reject_path_with_fragment":                                             false,
+		"envoy.restart_features.use_eds_cache_for_ads":                                                         true,
```

Comment:
Eventually this would allow us to remove the `AlwaysRespond` code in WatchedResources, right?

Reply:
Oh wait it only kicks in _after_ the initial_fetch_timeout... which we have set to infinite. Does this even do anything?

Reply:
Yes. That is correct. Currently it is no-op for us. But I plan to change the init_fetch_timeout for EDS to default (15s) and possibly remove AlwaysRespond code. Sorry - I should have mentioned in this PR. do you see any concerns with that?

Reply:
Goal is to mark "ready" only when we have initial state

Reply:
hmm. I thought it is safe to enable because cache is only used when a cluster is updated but not when it is added for the first time. I did not think about a case where initial fetch times out and cluster finishes warming with 0 endpoints. So looks like we need to two fetch timeouts or we need to set initFetchtime intelligently based on whether it is a new cluster vs. update.

Reply:
Just curious why do we even have a time for when to use the cache? Why not use the cache instantly, then if an update comes we can replace it?

Reply:
> Just curious why do we even have a time for when to use the cache? Why not use the cache instantly, then if an update comes we can replace it?

Yes. I think that makes sense. Let me think about it.